kickstart/provision.erb

```diff
@@ -143,6 +144,11 @@ echo "updating system time"
 # update all the base packages from the updates repository
 yum -t -y -e 0 update
 
+<% if chef_enabled %>
+echo "Bootstraping chef"
+<%= respond_to?(:chef_bootstrap) ? chef_bootstrap(@host) : snippet_if_exists("chef-client omnibus bootstrap") %>
```

Comment:
I understand, however, AFAIK changes to the templates can not be done by editing the seed files, you would have to add a migration in that case and you can't be sure what people are changing (but you could lock them probably). Havin the templates here in git and using foreman_templates is better, IMHO.

I leave it up to you :)